tests/test_wallet.py

```diff
@@ -177,5 +193,29 @@ def test_from_int(self):
         assert isinstance(key, PrivateKey)
         assert key.to_int() == PRIVATE_KEY_NUM
 
+    def test_from_hex_testnet(self):
```

Comment:
It would be good to add these tests for stn as well.

Reply:
Yep. Sure. That's no problem. :)

Reply:
I've been looking at this and there's really no point whatsoever in adding tests for stn... the addresses and one byte prefix is the exact same for test and stn... basically this would amount to adding new functions with a different name but doing the *exact* same thing with the *exact* same addresses etc. and executing the exact same code... no point imo.

Reply:
The only places it makes sense is where there might be a parameter that could be "main" "test" or "stn" and in those cases it is good to make sure that nothing breaks for the "stn" option... but for functions that are just raw cryptographic functions... it's all the same so no point testing twice imo

Reply:
That makes sense. It is a fine decision as long as it can be assumed that STN addresses will always be the same as testnet addresses.